src/routes.spec.js

```diff
@@ -1540,7 +1540,7 @@ describe('POST /api/convert/hl7/:template (with stored template)', function () {
 
     it('should return 200 OK with valid message and existing template in a subdirectory', function (done) {
         supertest(app)
-            .post("/api/convert/hl7/Resources/ADT_A01/Patient.hbs")
+            .post("/api/convert/hl7/Resources/Patient.hbs")
```

Comment:
Same question on this. What happens to the overall code if they Resources/Patient template is removed for some reason?

Reply:
same comment as above.